### PR TITLE
Dictation and grammar corrections for chapters 1-4

### PR DESCRIPTION
**Pull request by salmasian**
This includes several minor changes to grammar or dictation, as well as the addition of a link for the hashtag search on Twitter.

### CODE REVIEW

**Comment by salmasian** [comment]
CI seems to have failed for reasons unrelated to this change:

```
Quitting from lines 296-297 (intro.Rmd) 
Error in loadNamespace(name) : there is no package called 'devtools'
```

File: intro.Rmd [hunk 1]
```diff
@@ -95,7 +95,7 @@ It's common to think about modelling as a tool for hypothesis confirmation, and
 
 ## Prerequisites
 
-We've made a few assumptions about what you already know in order to get the most out of this book. You should be generally numerically literate, and it's helpful if you have some programming experience already. If you've never programmed before, you might find [Hands on Programming with R](http://amzn.com/1449359019) by Garrett to be a useful adjunct to this book.
+We've made a few assumptions about what you already know in order to get the most out of this book. You should be generally numerically literate, and it's helpful if you have some programming experience already. If you've never programmed before, you might find [Hands on Programming with R](http://amzn.com/1449359019) by Garrett Grolemund to be a useful adjunct to this book.
```
Comment (garrettgman) [inline intro.Rmd]: I disagree. We are writing in a friendly, conversational tone—one that invites the reader to be on a first name basis with the authors.

Comment (salmasian) [reply intro.Rmd]: Fine. I will undo this one.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**